home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -2615,4 +2924,11 @@ components:
       schema:
         $ref: '#/components/schemas/PollStatus'
       description: Status of the Poll
-      example: draft
\ No newline at end of file
+      example: draft
+    question_type:
+      in: query
+      name: type
+      schema:
+        $ref: '#/components/schemas/QuestionType'
+      description: Type of the Question
+      example: advice
```

Comment:
Can we add new line at the end of the file?

Reply:
Sure, added